fury-next-on-eks/README.md

```diff
@@ -0,0 +1,536 @@
+        opensearch:
+          type: single
+          resources:
+            limits:
+              cpu: 2000m
+              memory: 4G
+            requests:
+              cpu: 300m
+              memory: 1G
+      dr:
+        velero:
+          eks:
+            region: eu-west-1
+            bucketName: <S3_VELERO_BUCKET_NAME>
+```
+
+Replace the field `<S3_VELERO_BUCKET_NAME>` with the name of the S3 bucket that will be used to store the Velero backups.
+You can configure the existing modules or add new ones (take a look to the [docs][fury-distribution-eks-reference]) should you prefer.
+
+From these parameters, `furyctl` will automatically configure and deploy the battle-tested Kubernetes Fury Distribution.
+
+## Step 2 - Provisioning an EKS Cluster Automatically with furyctl
+
+In this section, you will utilize furyctl to automatically provision an EKS Cluster, making the deployment process streamlined.
+
+1. Start by running the furyctl command to create the cluster:
+
+```bash
+furyctl create cluster
+```   
+
+2. Upon being prompted to connect to the VPN, simply open the .ovpn file via your OpenVPN Client application.
+
+3. Connect to the OpenVPN Server via the chosen OpenVPN Client and continue by pressing `enter`
+
+4. Once connected to the VPN the process will continue to provision the cluster and the distribution.
+
+> ⏱ The process will take several minutes to complete, you can follow the progress in detail by running the following command:
+>
+> ```bash
+> tail -f /root/.furyctl/furyctl.log
+> ```
+
+🚀 Success! The distribution is fully deployed. Proceed to the next section to explore the various features it has to offer.
+
+## Step 3 - Explore the Fury Kubernetes Distribution
+
+### Setup local DNS
+
+In the previous section, alongside the distribution, you have deployed Kubernetes ingresses to expose underlying services at the following HTTP routes:
+
+- `directory.internal.demo.example.dev`
+- `gpm.internal.demo.example.dev`
+- `cerebro.internal.demo.example.dev`
+- `grafana.internal.demo.example.dev`
+- `opensearch-dashboards.internal.demo.example.dev`
+
+To access the ingresses more easily via the browser, configure your local DNS to resolve the ingresses to the internal load balancer IP:
+
+1. Get the IP address of the internal load balancer:
+
+```bash
+dig $(kubectl get svc ingress-nginx-internal -n ingress-nginx --no-headers | awk '{print $4}')
+```
+
+Output:
+
+```bash
+...
+
+;; ANSWER SECTION:
+xxx.elb.eu-west-1.amazonaws.com. 77 IN A <FIRST_IP>
+xxx.elb.eu-west-1.amazonaws.com. 77 IN A <SECOND_IP>
+xxx.elb.eu-west-1.amazonaws.com. 77 IN A <THIRD_IP>
+...
+
+```
+
+3. Add the following line to your machine's `/etc/hosts` (not the container's):
+
+```bash
+<FIRST_IP> directory.internal.demo.example.dev cerebro.internal.demo.example.dev opensearch-dashboards.internal.demo.example.dev grafana.internal.demo.example.dev gpm.internal.demo.example.dev
+```
+
+Now, you can reach the ingresses directly from your browser.
```

Comment:
```suggestion
Now, you can reach the ingresses directly from your browser.

> 💡 **Advanced**: if you have a delegated zone already configured in Route53, you can use a subdomain of that zone instead of `demo.example.dev` and the DNS should just work.

```